### PR TITLE
completed string-manipulation-2

### PR DESCRIPTION
**Pull request by PhilChoe**
image
image

### CODE REVIEW

File: string-manipulation-2/src/swap-chars.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* exported swapChars */
+
+function swapChars(
```
Comment (hesam) [inline string-manipulation-2/src/swap-chars.ts:3]: You're using advanced syntax we haven't covered. There's a simpler solution: a simple for loop to go over each index of input, and using if/else conditionals to check we're looking at an index which needs to be swapped.

**Review by hesam — APPROVED** [review]
Please see the comment in code and try to see if you can work the simpler solution.

**Comment by PhilChoe** [comment]
> Please see the comment in code and try to see if you can work the simpler solution.

thank you for the hint (:

**Review by hesam — CHANGES_REQUESTED** [review]
For swapChars your new solution is almost correct, but you're missing the case if for loop index i is neither firstIndex nor secondIndex... So at the end of conditional you need an else branch such as these: 


else {
result += string[i]
}

**Review by hesam — COMMENTED** [review]
yep 👍 💯

**Pull request merged**